main.js

```diff
@@ -9,11 +9,29 @@ const platform      = require('os').platform();
 const rxIpc         = require('rx-ipc-electron/lib/main').default;
 const Observable    = require('rxjs/Observable').Observable;
 const log           = require('electron-log');
-
-log.transports.file.appName = (process.platform == 'linux' ? '.particl' : 'Particl');
-log.transports.file.file = log.transports.file
+/*
+ * Logger config
+ */
+log.transports.file.appName  = (process.platform == 'linux' ? '.particl' : 'Particl');
+log.transports.file.file     = log.transports.file
    .findLogPath(log.transports.file.appName)
    .replace('log.log', 'particl.log');
+
+log.transports.console.level = 'info';
+log.transports.file.level = 'debug';
+
+if (process.argv.includes('-v')) {
+  log.transports.console.level = 'debug'; /* most messages */
+  process.argv.push('-printtoconsole');   /* daemon output */
```

Comment:
Can we maybe put the logging to console defined by another argument?
I'd like to be able to get general debug messages (Node backend), without the core logs.

Reply:
Absolutely, thanks for noticing. Here's what I propose : 
-v : gui[debug]
-vv : gui[debug] + daemon[normal]
-vvv : gui[silly] + daemon[debug]

Reply:
@kewde